source/_integrations/aidot.markdown

```diff
@@ -0,0 +1,25 @@
+---
+title: Aidot
+description: Instructions on integrating Aidot home security with Home Assistant.
+ha_category:
+  - Light
+ha_release: 2024.6
+ha_iot_class: local_polling
+ha_config_flow: true
+ha_codeowners:
+  - '@sz411'
```

Comment:
```suggestion
  - '@s1eedz'
```
Was sz411 listed here on purpose? The core PR has your name on it. If you are the code owner, your name should be here.